libs/tup-components/src/projects/ProjectsTable.tsx

```diff
@@ -18,9 +20,11 @@ const allocationDisplay = (allocations: ProjectsAllocations[]) => {
 
 export const ProjectsTable: React.FC = () => {
   const { data, isLoading, error } = useProjects();
-  const projectData = data?.filter((prj) =>
-    prj.allocations?.some((alloc) => alloc.status === 'Active')
-  );
+  const projectData = data
+    ?.filter((prj) =>
+      prj.allocations?.some((alloc) => alloc.status === 'Active')
+    )
+    .slice(0, PROJECTS_DASHBOARD_DISPLAY_LIMIT);
```

Comment:
Added constant[^1] and used it to limit projects shown (cuz this table already has a "View all …" link).

[^1]: Just like in `libs/tup-components/src/tickets/TicketsTable.tsx`.